experiments/AMIP/components/atmosphere/climaatmos_init.jl

```diff
@@ -81,6 +81,9 @@ function atmos_init(::Type{FT}, atmos_config_dict::Dict) where {FT}
     integrator.p.precomputed.sfc_conditions.ρ_flux_uₕ.components .= Ref(SMatrix{1, 2}([FT(0), FT(0)]))
     integrator.p.precipitation.col_integrated_rain .= FT(0)
     integrator.p.precipitation.col_integrated_snow .= FT(0)
+    integrator.p.precipitation.ᶜS_ρq_tot .= FT(0)
+    integrator.p.precipitation.ᶜ3d_rain .= FT(0)
+    integrator.p.precipitation.ᶜ3d_snow .= FT(0)
```

Comment:
are these new fields in the atmos integrator? or were we just not using them before?

Reply:
They are, but they are not initialized to zero when we first call the precipitation energy check (implemented in this PR).